### PR TITLE
Fix redis namespace

### PR DESCRIPTION
**Pull request by maksimf**

### CODE REVIEW

File: lib/splitclient-rb/split_config.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def initialize(opts = {})
       @events_uri = (opts[:events_uri] || SplitConfig.default_events_uri).chomp('/')
       @mode = opts[:mode] || SplitConfig.default_mode
       @redis_url = opts[:redis_url] || SplitConfig.default_redis_url
-      @redis_namespace = opts[:redis_namespace] || SplitConfig.default_redis_namespace
+      @redis_namespace = opts[:redis_namespace] ? "#{opts[:redis_namespace]}/#{SplitConfig.default_redis_namespace}" : SplitConfig.default_redis_namespace
```
Comment (patricioe) [inline lib/splitclient-rb/split_config.rb]: are we adding a slash? 

@sarrubia

Comment (patricioe) [inline lib/splitclient-rb/split_config.rb]: My understanding according to the spec is: `userPrefix.SPLITIO.blablabla` and i don't see that here.

Comment (maksimf) [inline lib/splitclient-rb/split_config.rb]: @patricioe @sarrubia 
There is a little ambiguity in the [docs](https://docs.google.com/document/d/10NSU3Fgmfa74ElM2mZGTqdNjwPG5mHmd5XSJG8X3nkA/edit), both `/` and `.` is mentioned there. What should I use?

**Comment by maksimf** [comment]
@patricioe We're testing name space as well now

File: lib/splitclient-rb/split_config.rb (hunk 1 above)
Comment (sarrubia) [inline lib/splitclient-rb/split_config.rb]: @maksimf @patricioe we are using a mix between dot and slash. Let me show a sample pattern to clarify it.

`userPrefix.SPLITIO/{sdk-language-version}/{instance-id}/impressions.{featureName}`

As you can see the slash is used to separate the `sdk-language-version` and the` instance-id`

**Pull request merged**